Models/TrainingProgram.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace BangazonAPI.Models
+{
+    public class TrainingProgram
+    {
+        [Key]
+        public int TrainingProgramId { get; set; }
+
+        [Required]
+        public string ProgramName { get; set; }
+        [StringLength(80,
```

Comment:
The data annotation should come before the property- so if you want this to apply to `ProgramName`, it needs to go above line 14